### PR TITLE
Extend metrics

### PR DESCRIPTION
**Pull request by oribon**
Depends on https://github.com/metallb/frr-k8s/pull/33
this aligns the ports to not clash with MetalLB and adds some metrics

### CODE REVIEW

**Comment by fedepaol** [comment]
just align the helm charts too
All the rest looks fine

**Comment by oribon** [comment]
> just align the helm charts too All the rest looks fine

done

File: charts/frr-k8s/templates/controller.yaml [hunk 1]
```diff
@@ -188,7 +188,7 @@ spec:
         args:
         - "--node-name=$(NODE_NAME)"
         - "--namespace=$(NAMESPACE)"
-        - "--metrics-bind-address=127.0.0.1:{{ .Values.prometheus.metricsPort }}"
+        - "--metrics-bind-address=:{{ .Values.prometheus.metricsPort }}"
```
Comment (fedepaol) [inline charts/frr-k8s/templates/controller.yaml:191]: I think binding to 127.0.0.1 is better than listening on any

Comment (oribon) [reply charts/frr-k8s/templates/controller.yaml:191]: I changed it here so it is aligned with the regular manifests and the code's default (when we generated the controller), do you think we should change it everywhere?

Comment (fedepaol) [reply charts/frr-k8s/templates/controller.yaml:191]: yeah I think this is better as the pod is hostnetworked adn we won't expose the port outside.

**Comment by fedepaol** [comment]
LGTM

**Pull request merged**